### PR TITLE
Read wishlist

### PR DESCRIPTION
**Pull request by rahilkadakia**
#17 
1. Added the `read_wishlists` GET endpoint.
2. Added happy and sad test paths for `read_wishlists`.
3. Added `test_bad_request` test to maximize coverage.
4. Added `test_method_not_allowed` test to maximize coverage.
5. Added `_create_wishlists` to create dummy wishlists (to read from , while testing `test_get_wishlist`).
6. Commented `resource_conflict` from `/service/common/error_handlers.py` since we may not use it (as per Prof on Slack).
7. `internal_server_error` still to be implemented - not covered.
8. Code Coverage: 97%

### CODE REVIEW

File: service/common/error_handlers.py
Comment (studentgrad0419) [inline service/common/error_handlers.py:1]: looks good if we're not using it.

File: service/common/error_handlers.py [hunk 1]
```diff
@@ -71,19 +71,19 @@ def method_not_supported(error):
...
+#     message = str(error)
+#     app.logger.warning(message)
+#     return (
+#         jsonify(
+#             status=status.HTTP_409_CONFLICT,
+#             error="Conflict",
+#             message=message,
+#         ),
+#         status.HTTP_409_CONFLICT,
+#     )
```
Comment (studentgrad0419) [inline service/common/error_handlers.py:86]: looks good if we're not using it.

Comment (rahilkadakia) [reply service/common/error_handlers.py:86]: yeah, i don't think we will. we can always uncomment it in the further iterations if needed.

**Review by studentgrad0419 — APPROVED** [review]
Looks good to merge if green and make lint is good. Nice job with the happy and sad path inclusions.

**Pull request merged**